tests/fromRdf-manifest.jsonld

```diff
@@ -379,6 +379,22 @@
       "input": "fromRdf/js11-in.nq",
       "expect": "fromRdf/js11-out.jsonld",
       "option": {"specVersion": "json-ld-1.1"}
+    }, {
+      "@id": "#tjs12",
+      "@type": ["jld:PositiveEvaluationTest", "jld:FromRDFTest"],
+      "name": "JSON literals (strings)",
+      "purpose": "Tests creating property with multiple rdf:type rdf:JSON to a JSON literal (null).",
+      "input": "fromRdf/js12-in.nq",
+      "expect": "fromRdf/js12-out.jsonld",
+      "option": {"specVersion": "json-ld-1.1"}
+    }, {
+      "@id": "#tjs13",
+      "@type": ["jld:PositiveEvaluationTest", "jld:FromRDFTest"],
+      "name": "Mixed JSON literals and non-JSON",
+      "purpose": "Tests creating property with mixed rdf:type rdf:JSON and non-JSON.",
```

Comment:
```suggestion
      "name": "Mixed JSON literal and non-JSON",
      "purpose": "Tests creating property with mixed values: literal of datatype rdf:type and non-JSON.",
```

Actually, the test is mixing a literal of datatype `rdf:JSON` and an IRI, so the text is still a bit misleading (I would expect "non-JSON" to mean "a literal with another datatype than `rdf:JSON`).